frontend/src/app/components/audit/audit.component.css

```
@@ -215,3 +230,447 @@ td.mat-cell {
+		flex-direction: column;
+		gap: 10px;
+	}
+
+	.table-wrapper tr.mat-mdc-row {
+		background: var(--mat-table-background-color, #fff);
+		border-radius: 12px;
+		padding: 10px 14px;
+		box-shadow: 0 1px 2px rgba(0, 0, 0, 0.06), 0 1px 4px rgba(0, 0, 0, 0.04);
+		display: grid !important;
+		grid-template-columns: auto 1fr;
+		grid-column-gap: 12px;
```

Stylelint (17.12.0)

[error] Expected "grid-column-gap" to be "column-gap" (property-no-deprecated)

(property-no-deprecated)